pkgs/applications/misc/kuro/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, fetchFromGitHub
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, mkYarnPackage
+, electron
+}:
+
+mkYarnPackage rec {
+  pname = "kuro";
+  version = "9.0.0";
+  executableName = pname;
+
+  src = fetchFromGitHub {
+    owner = "davidsmorais";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-9Z/r5T5ZI5aBghHmwiJcft/x/wTRzDlbIupujN2RFfU=";
+  };
+
+  packageJSON = ./package.json;
+  yarnLock = ./yarn.lock;
+  yarnNix = ./yarn.nix;
+
+  nativeBuildInputs = [
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    # resources
+    mkdir -p "$out/share/kuro/electron"
+    cp -r './deps/kuro/static' "$out/share/kuro/electron"
+    cp -r './deps/kuro/src' "$out/share/kuro/electron"
+    cp -r './deps/kuro/index.js' "$out/share/kuro/electron"
+    cp -r './node_modules' "$out/share/kuro/electron"
```

Comment:
Can we just use `electron-builder`?

Reply:
Ok, I have switched to electron-builder now